docs/examples/kubo-as-a-library/go.mod

```diff
@@ -7,7 +7,7 @@ go 1.18
 replace github.com/ipfs/kubo => ./../../..
 
 require (
-	github.com/ipfs/go-libipfs v0.1.0
+	github.com/ipfs/go-libipfs v0.1.1-0.20230114214320-c9e692f8ce8f
```

Comment:
TODO: we need to release https://github.com/ipfs/go-libipfs/pull/31 and switch to release version before merging this PR.